rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -93,6 +94,42 @@
 MIN_LOGS_PROCESSED_TO_SLEEP = 1000
 
 
+def decode_safely(
+        msg_aggregator: 'MessagesAggregator',
+        tx_hash: EVMTxHash,
+        chain_id: ChainID,
+        func: Callable,
+        *args: tuple[Any],
+        **kwargs: Any,
+) -> tuple[Any, bool]:
+    """
+    Wrapper for methods that execute logic from decoders. It handles all known errors
+    by logging them and optionally sending them to the user.
```

Comment:
I said that if you change the optionality you needed to ofc also change the docstring

```suggestion
    by logging them and sending them to the user.
```